app/themes/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Ideally, for the new folders, we should use the `.eslintrc` from `viz`.

That one extends the airbnb eslint base, and denotes that we use `single` for our quotes rule.  If there are sections within the theme that require a different rule (because it wouldn't make sense to use single quotes), then we can disable or modify the rule via an `eslint-disable` comment within the code.

Reply:
I've moved over a copy of the viz `.eslintrc` to the `elements` folder for the new components, but I did not see (or find with search) a quotes rule in viz, there is one already in the blip main `.eslintrc` -- seems to be working well when running it, but being mindful of the proliferation of `.eslintrc` files I removed the one specific to the `baseTheme` and just disabled the quote rule with a `eslint-disable` comment at the top of the file

Reply:
The quotes rule comes from the airbnb eslint config that the one from viz extends.  Basically, we use that one as a starting point and then override it as required for our preferences.